src/composables/aeSdk.ts

```diff
@@ -204,6 +210,17 @@ export function useAeSdk() {
     );
   }
 
+  async function disconnectDapps() {
+    if (IN_FRAME) {
+      const aeSdkLocal = await getAeSdk();
+      aeSdkLocal._clients.forEach((aepp) => {
+        if (aepp.rpc.connection.isConnected()) {
+          aepp.rpc.connection.disconnect();
+        }
+      });
```

Comment:
@davidyuk do you think this is a good way of disconnecting dapps?

Reply:
The way it is currently, it would just unbind event listeners without notifying the other peer
https://github.com/aeternity/aepp-sdk-js/blob/315718fbc34ab92df8b9fc04bf186d00f2af7151/src/aepp-wallet-communication/connection/BrowserWindowMessage.ts#L88-L97

```suggestion
      aeSdkLocal._clients.forEach((aepp, aeppId) => {
        // if aepp didn't disconnected before by itself
        if (aepp.status === RPC_STATUS.CONNECTED) {
          // notify aepp that wallet disconnects
          aepp.rpc.notify(METHODS.closeConnection, null);
        }
        // internally it is closes connection and removes aepp from `_clients`
        this.aeSdk.removeRpcClient(aeppId);
      });
```
Based on https://github.com/aeternity/aepp-sdk-js/blob/315718fbc34ab92df8b9fc04bf186d00f2af7151/examples/browser/wallet-iframe/src/App.vue#L117-L126
Also, ensure that is works the same as https://docs.aeternity.com/aepp-sdk-js/v13.2.2/examples/browser/wallet-iframe/, compatible with https://docs.aeternity.com/aepp-sdk-js/v13.2.2/examples/browser/aepp/

Reply:
Thank you very much. I updated the code with you suggestions.